refs/biblio.json

```diff
@@ -4211,6 +4218,20 @@
         "date": "2008",
         "isoNumber": "ISO 639-5:2008"
     },
+    "ITU-R-BT.601": {
+        "href": "https://www.itu.int/rec/R-REC-BT.601/",
+        "title": "BT.601 : Studio encoding parameters of digital television for standard 4:3 and wide screen 16:9 aspect ratios",
```

Comment:
Last I heard, ITU expects the reference to be displayed as:

_Recommendation ITU-R BT.601-7 (03/2011), Studio encoding parameters of digital television for standard 4:3 and wide-screen 16:9 aspect ratios_

Reply:
done